public/editor/scripts/main.js

```diff
@@ -52,6 +52,16 @@ require(["jquery", "bowser"], function($, bowser) {
 
   Bramble.once("error", onError);
 
+  var errorMessageTimeoutMS = 10000;
+
+  setTimeout(function(){
+    showLoadingErrorMessage();
+  }, errorMessageTimeoutMS);
```

Comment:
Can you add analytics here so we record that this happened?  Would be good to know how often we hit it.